docs/index.asciidoc

```diff
@@ -1,5 +1,8 @@
 = APM Python Agent Reference
 
+:branch: current
+include::{asciidoc-dir}/../../shared/attributes.asciidoc[]
```

Comment:
Excuse my ignorance, but where does `asciidoc-dir` come from? Is it like a global variable?